includes/CMB2_hookup.php

```diff
@@ -369,17 +374,73 @@ public function register_column_headers( $columns ) {
 	 */
 	public function column_display( $column_name, $object_id ) {
 		if ( isset( $this->columns[ $column_name ] ) ) {
-				$field = new CMB2_Field( array(
-					'field_args'  => $this->columns[ $column_name ]['field'],
-					'object_type' => $this->object_type,
-					'object_id'   => $this->cmb->object_id( $object_id ),
-					'cmb_id'      => $this->cmb->cmb_id,
-				) );
+			$field = new CMB2_Field( array(
+				'field_args'  => $this->columns[ $column_name ]['field'],
+				'object_type' => $this->object_type,
+				'object_id'   => $this->cmb->object_id( $object_id ),
+				'cmb_id'      => $this->cmb->cmb_id,
+			) );
+
+			$this->cmb->get_field( $field )->render_column();
+		}
+	}
+
+	/**
+	 * Returns the columns sortable array.
+	 *
+	 * @since 2.6.1
+	 *
+	 * @param array  $columns An array of sortable columns.
+	 */
+	public function columns_sortable( $columns ) {
+		$fields = $this->cmb->prop( 'fields' );
 
-				$this->cmb->get_field( $field )->render_column();
+		foreach ( $fields as $key => $field ) {
+			if ( ! isset( $field['column'] ) ) {
+				continue;
+			}
+			$columns[ $field['id'] ] = $field['id'];
 		}
+
+		return $columns;
 	}
 
+	/**
+	 * Return the order by on custom columns
+	 *
+	 * @since 2.6.1
+	 * 
+	 * @param array  $columns An array of sortable columns.
+	 */
+	public function columns_sortable_orderby( $query ) {
+		if( ! is_admin() ){
```

Comment:
Please clean up others similar to this (if there are others)
```suggestion
		if ( ! is_admin() ) {
```